### PR TITLE
DOCS: Proofreading Samples C++ - port to master

### PR DESCRIPTION
**Pull request by msmykx-intel**
Proofreading of C++ Samples

Port from https://github.com/openvinotoolkit/openvino/pull/14184

### CODE REVIEW

File: samples/cpp/classification_sample_async/README.md [hunk 1]
```diff
@@ -14,36 +14,41 @@ The following C++ API is used in the application:
...
 
-Then, the sample creates an inference request object and assigns completion callback for it. In scope of the completion callback
+Next, the sample creates an inference request object and assigns completion callback for it. In scope of the completion callback,
 handling the inference request is executed again.
 
-After that, the application starts inference for the first infer request and waits of 10th inference request execution being completed. The asynchronous mode might increase the throughput of the pictures.
+After that, the application starts inference for the first infer request and waits until the 10th inference request execution is completed. The asynchronous mode might increase the throughput of the pictures
 
-When inference is done, the application outputs data to the standard output stream. You can place labels in .labels file near the model to get pretty output.
+When inference is done, the application outputs data to the standard output stream. Labels may placed in `.labels` file near the model to get pretty output.
```
Comment (zKulesza) [inline samples/cpp/classification_sample_async/README.md:36]: Grammer error: 
Labels may be placed in the '.labels' file near the model to get a prettier output. 

Is pretty necessary here?

File: samples/cpp/hello_classification/README.md [hunk 2]
```diff
@@ -17,39 +17,40 @@ The following C++ API is used in the application:
...
 | Validated Models | [alexnet](@ref omz_models_model_alexnet), [googlenet-v1](@ref omz_models_model_googlenet_v1) |
-| Model Format | OpenVINO™ toolkit Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
+| Model Format | OpenVINO Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
 | Supported devices | [All](../../../docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md) |
 | Other language realization | [C](../../../samples/c/hello_classification/README.md), [Python](../../../samples/python/hello_classification/README.md) |
 
 ## How It Works
 
-At startup, the sample application reads command line parameters, prepares input data, loads a specified model and image to the OpenVINO™ Runtime plugin and performs synchronous inference. Then processes output data and write it to a standard output stream.
+At startup, the sample application reads command-line parameters, prepares input data, loads a specified model and image to the OpenVINO Runtime plugin and performs synchronous inference. Then processes output data and write it to a standard output stream.
```
Comment (zKulesza) [inline samples/cpp/hello_classification/README.md:26]: output data and writes it to

File: samples/cpp/hello_nv12_input_classification/README.md [hunk 3]
```diff
@@ -10,65 +10,68 @@ The following C++ API is used in the application:
...
+| Validated images | An uncompressed image in the `NV12` color format - \*.yuv |
 | Supported devices | [All](../../../docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md) |
 | Other language realization | [C](../../../samples/c/hello_nv12_input_classification/README.md) |
 
 ## How It Works
 
-At startup, the sample application reads command line parameters, loads the specified model and an
-image in the NV12 color format to an OpenVINO™ Runtime plugin. Then, the sample creates an synchronous inference request object. When inference is done, the application outputs data to the standard output stream. You can place labels in .labels file near the model to get pretty output.
+At startup, the sample application reads command-line parameters, loads the specified model and an
+image in the `NV12` color format to an OpenVINO Runtime plugin. Then, the sample creates an synchronous inference request object. When inference is done, the application outputs data to the standard output stream. You can place labels in `.labels` file near the model to get pretty output.
```
Comment (zKulesza) [inline samples/cpp/hello_nv12_input_classification/README.md:26]: Then, the sample creates a synchronous inference request object.

File: samples/cpp/hello_reshape_ssd/README.md [hunk 4]
```diff
@@ -12,64 +12,65 @@ The following C++ API is used in the application:
...
 | Validated Models | [person-detection-retail-0013](@ref omz_models_model_person_detection_retail_0013) |
-| Model Format | OpenVINO™ toolkit Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
+| Model Format | OpenVINO Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
 | Supported devices | [All](../../../docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md) |
 | Other language realization | [Python](../../../samples/python/hello_reshape_ssd/README.md) |
 
 ## How It Works
 
-Upon the start-up the sample application reads command line parameters, loads specified network and image to the Inference
+Upon the start-up the sample application reads command-line parameters, loads specified network and image to the Inference
```
Comment (zKulesza) [inline samples/cpp/hello_reshape_ssd/README.md:26]: loads the specified network

File: samples/cpp/speech_sample/README.md [hunk 5]
```diff
@@ -136,7 +136,7 @@ The following pre-trained models are available:
 - rm_lstm4f
 - rm_cnn4a_smbr
 
-All of them can be downloaded from [https://storage.openvinotoolkit.org/models_contrib/speech/2021.2](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2).
+All of them can be downloaded from the [storage](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2).
```
Comment (zKulesza) [inline samples/cpp/speech_sample/README.md:139]: All of them can be downloaded from [storage]

File: samples/cpp/speech_sample/README.md [hunk 6]
```diff
@@ -228,9 +228,9 @@ Kaldi's nnet-forward command. Since the `speech_sample` does not yet use pipes,
    cat out.txt | utils/int2sym.pl -f 2- words.txt | sed s:\<UNK\>::g | compute-wer --text --mode=present ark:test_filt.txt ark,p:-
    ```
 
-   All of mentioned files can be downloaded from [https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/wsj_dnn5b_smbr](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/wsj_dnn5b_smbr)
+   All of mentioned files can be downloaded from the [storage](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/wsj_dnn5b_smbr)
```
Comment (zKulesza) [inline samples/cpp/speech_sample/README.md:231]: All of the mentioned files can be downloaded from [storage]

**Review by zKulesza — CHANGES_REQUESTED** [review]
Added comments where small changes are necessary.

File: samples/cpp/speech_sample/README.md (hunk 6 above)
Comment (msmykx-intel) [reply samples/cpp/speech_sample/README.md]: Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0

File: samples/cpp/speech_sample/README.md (hunk 5 above)
Comment (msmykx-intel) [reply samples/cpp/speech_sample/README.md]: Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0

File: samples/cpp/hello_reshape_ssd/README.md [hunk 7]
```diff
@@ -12,64 +12,65 @@ The following C++ API is used in the application:
...
 Engine plugin. Then, the sample creates an synchronous inference request object. When inference is done, the application creates output image and output data to the standard output stream.
```
[the remark this answers is not in the document]
Comment (msmykx-intel) [reply samples/cpp/hello_reshape_ssd/README.md]: Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0

File: samples/cpp/hello_nv12_input_classification/README.md (hunk 3 above)
Comment (msmykx-intel) [reply samples/cpp/hello_nv12_input_classification/README.md]: Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0

File: samples/cpp/hello_classification/README.md (hunk 2 above)
Comment (msmykx-intel) [reply samples/cpp/hello_classification/README.md]: Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0

File: samples/cpp/classification_sample_async/README.md (hunk 1 above)
Comment (msmykx-intel) [reply samples/cpp/classification_sample_async/README.md]: Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0

As for "pretty" - We can change it to something like:
"Labels may be placed in the '.labels' file near the model to get a better looking output." 
or "Labels may be placed in the '.labels' file near the model to make an output look cleaner".

Comment (zKulesza) [reply samples/cpp/classification_sample_async/README.md]: "Labels may be placed in the '.labels' file near the model to make an output look cleaner".

This sounds a lot better. I would change it to this.

Comment (msmykx-intel) [reply samples/cpp/classification_sample_async/README.md]: Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/ee9ed62f53be4cfb2fa894f36dd3742765bf7dec

**Review by zKulesza — APPROVED** [review]

**Comment by msmykx-intel** [comment]
Proofreading for master and 22.3 will be done separately after the 22.3 release.

**Pull request closed without merging**